pkgs/development/node-packages/overrides.nix

```
@@ -40,6 +40,13 @@ final: prev: {
     };
   };
 
+  bash-language-server = prev.bash-language-server.override {
+    nativeBuildInputs = [ pkgs.buildPackages.makeWrapper ];
+    postInstall = ''
+      wrapProgram "$out/bin/bash-language-server" --prefix PATH : ${lib.makeBinPath [ pkgs.shellcheck ]}
```

```suggestion
      wrapProgram "$out/bin/bash-language-server" \
        --prefix PATH : ${lib.makeBinPath [ pkgs.shellcheck ]}
```
looks nicer to me but that's just personal preference.

---

Yeah I agree on the formatting! I rebased and force pushed so that it's under the same commit. Thanks